interviews/internships.md

```diff
@@ -0,0 +1,84 @@
+# Internships
+
+Tilt partners with [HackNY](https://hackny.org/) to offer a summer engineering
+internship.
+
+As a small startup, our internship program looks different than interships at
+more established companies.
+
+## Goals
+
+- An intern at Tilt is a full part of a small startup team.
+
+- The intern should get the opportunity to meet everyone on the team and 
+  learn about what they do day-to-day.
+
+- Conversely, everyone on the team should have the opportunity to pair with the
+  intern and help them grow. The ability to learn and teach is an
+  important part of being an engineer.
+  
+## Non-Goals
+
+- At some companies, a successful internship can turn into a full-time offer.
+  At Tilt, this is a non-goal. Small startup hiring is highly contrained, and
+  it's simply unlikely that our hiring needs will line up with intern hiring
+  timelines. Tilt has never made a full-time offer to an intern. But we have
+  used our networks to help them find their next job.
+  
+- Some companies try to hire interns for a particular skillset or short-term project.
+  For example, they might hire an AI PhD for a predefined AI project. We don't need
+  Tilt interns to have prior experience in Kubernetes, infra, or devtools.
+
+## Corollaries
+
+- We don't have a pool of side projects that are designated "intern"
+  projects. Interns at Tilt will collaborate with other Tilters on our regular
+  feature backlog. A full-time Tilt engineer will help the intern scope work to
+  a realistic size.
+
+- We don't expect interns to have worked with Kubernetes or multi-service development
+  before they join. But we do expect interns to have some desire to learn Kubernetes
+  and experiment with it.
+
+## What We Look For
+
+We use our normal [engineering rubric](engineering.md) to evaluate intern
+candidates. But there are a few qualities we pay particular attention to in
+interns:
+
+- Prior job experience: This might be a previous internship, or a previous
+  job. An internship at Tilt is not a good first internship.
+
+- Prior project experience: This might be experience working with Go (or a
+  Go-like language), or experience working in a microservice-based architecture.
+  Having something familiar to anchor against can help with feeling overwhelmed,
+  since the intern will be learning a lot in a short time!
+
+- Autonomy/Resoluteness: A school project has a lot of structure (a syllabus,
+  deadlines, clear instructions, known solutions, etc.).  An engineering project
+  at a large company has less structure, but still might have multi-year
+  roadmaps. At a small startup, you might not know what you're working on next
+  month. This is not a good internship for a student who's looking for a lot of
+  predictability. So we look for demonstrated ability to work independently and
+  push through failed attempts.
```

Comment:
might be good to clarify that this bullet point is not about the ability to do it all yourself, but the ability to push the project forward even via (especially via) asking for help and pairing -- is that a good summary? (B/c that was the point I was most unsure about.)

Reply:
i really like this wording! done